metrics-delivery-endpoint-impl/src/main/resources/application.properties

```diff
@@ -9,3 +9,7 @@ counter.range.start=1609459200
 #Spring boot actuator
 management.endpoints.web.base-path=/-
 management.endpoints.web.exposure.include=health,info
+
+eosc.accounting-api.aai-token-url=https://aai-demo.eosc-portal.eu/auth/realms/core/protocol/openid-connect/token
+eosc.accounting-api.token-refresh-rate = 1800000
+eosc.accounting-api.push-rate = 86400000
```

Comment:
Regarding `*-rate` naming convention: I can see those are the millis but only thanks to this `*.properties` file. It will much more self-descriptive if the property name included the unit (e.g. with `ms` as suffix).

Reply:
OK. Also I think the readme now needs a section on application properties.

Reply:
Although the parameters in Spring also don't use units in their names, so maybe just write "ms" in the README.